libcontainer/specconv/spec_linux.go

```diff
@@ -190,6 +190,23 @@ func CreateLibcontainerConfig(opts *CreateOpts) (*configs.Config, error) {
 
 	exists := false
 	for _, m := range spec.Mounts {
+		// check mount options for (b)bind type mount
+		if m.Type == "bind" || m.Type == "rbind" {
+			if len(m.Options) == 0 {
+				return nil, fmt.Errorf("no options specified for %v type mount: [destination=%v, source=%v].", m.Type, m.Destination, m.Source)
+			} else {
+				blBind := false
+				for _, opt := range m.Options {
+					if opt == "bind" || opt == "rbind" {
+						blBind = true
+						break
+					}
+				}
+				if !blBind {
+					return nil, fmt.Errorf("options should contain (r)bind for %v type mount: [destination=%v, source=%v].", m.Type, m.Destination, m.Source)
```

Comment:
same

Reply:
use "logrus.Warnf" now. But we need `runc --log /dev/stdout` to display warn message, otherwise we just can see the error message `container_linux.go:344: starting container process caused "process_linux.go:424: container init caused \"rootfs_linux.go:58: mounting \\\"/etc/test\\\" to rootfs \\\"/opt/mygo/src/github.com/opencontainers/runctest/rootfs\\\" at \\\"/opt/mygo/src/github.com/opencontainers/runctest/rootfs/etc/test\\\" caused \\\"no such device\\\"\""`.